packages/tests/shadydom/shady-dynamic.html

```diff
@@ -2906,6 +3105,352 @@
           document.body.removeChild(d);
         });
       });
+
+      suite('querySelectorImplementation', function () {
```

Comment:
* Make sure to also add asserts for `querySelector` in addition to `querySelectorAll`.

Reply:
Ok, I added them for the tests for the tree-walking parts, but I left them out of the `parseSelectorList` tests since those are only to check if they parsed properly.